src/interface_r/scripts/gen_wrapper_utils.R

```diff
@@ -46,7 +48,18 @@ gen_wrapper <- function(
             param <- paste0("as_nullable_integer(", param, ")")
           }
           suffix <- ifelse(i < length(params), ",", "\n  )")
-          write_line(paste0("    ", params[[i]], " = ", param, suffix))
+          # if(param == "backend"){
+          #   write_line(paste0("    ", params[[i]], " = ", '"h2o4gpu"', suffix))
+          # } else 
```

Comment:
Are these safe to be removed as well?

Reply:
Yes, we want to expose the `backend` param to users.